### DISCUSSION TITLE
Popular & Trending Trakt lists search strings

### DISCUSSION THREAD

**Discussion by Omarscomin** [umbrellaplug/umbrellaplug.github.io#316] [Q&A]
Hi @umbrellaplug😊I hope you're doing well. Thanks again for all of your hard work with the most recent update. Everything is working great!

I'm wanting to configure my AH2 & Umbrella set up to be able to search Trakt lists. It would be nifty to be able to type "best of all time" into the AH2 search bar, and get some Trakt lists in the search results. Would you be willing & able to share Umbrella's search strings for popular and trending Trakt lists?

Thank you so much for all of your help.

**Comment by umbrellaplug** [comment]
@Omarscomin This would actually need a bit of a code change to work. The search lists function has a call to pull up the keyboard for input. I'll look at the code, it will need to be changed to allow for values to be passed in from the search.

**Reply by Omarscomin** [reply]
Please don't worry about it then, unless you genuinely _want_ to do it

**Comment by umbrellaplug** [comment]
posting these here so they're available.

`plugin://plugin.video.umbrella/?action=trakt_search_lists&media_type=tvshows&name=`

`plugin://plugin.video.umbrella/?action=trakt_search_lists&media_type=movies&name=`

**Reply by Omarscomin** [reply]
Thank you, UD! Can't wait to try this out later!
